cognite/client/data_classes/time_series.py

```diff
@@ -73,6 +73,44 @@ def plot(
             df.plot(*args, **kwargs)
             plt.show()
 
+    def count(self) -> int:
+        """Returns the number of datapoints in this time series.
+
+        This result may not be completely accurate, as it is based on aggregates which may be occasionally out of date.
+
+        Returns:
+            int: The number of datapoints in this time series.
+        """
+        identifier = utils._auxiliary.assert_at_least_one_of_id_or_external_id(self.id, self.external_id)
+        dps = self._cognite_client.datapoints.retrieve(
+            start=0, end="now", aggregates=["count"], granularity="10d", **identifier
```

Comment:
Why 10?

Reply:
I guess I just had to pick something. 10d * 10000 dps limit = 273 years, so then we will always get the counts in a single request.